dirsrvtests/tests/suites/indexes/regression_test.py

```diff
@@ -31,6 +34,8 @@
 BENAME2 = 'be2'
 
 DEBUGGING = os.getenv("DEBUGGING", default=False)
+logging.getLogger(__name__).setLevel(logging.INFO)
+log = logging.getLogger(__name__)
```

Comment:
It's added but not used. It might be worth rewriting some code comments to be log.info messages. :)